### ISSUE TITLE
Create automated integration tests for inbound business requirements

### ISSUE THREAD

**Issue by joeniquette** [department-of-veterans-affairs/va.gov-team#31466]
The task requires capybara tests be created for inbound integration testing at its most basic level. The intent is that we will iterate on creating more tests for a more robust integration suite over the next 6 months. The table below lists the initial requirements for this specific task. Future iterations are in the [alerting and monitoring epic](https://app.zenhub.com/workspaces/vsp-identity-5f5bab705a94c9001ba33734/issues/department-of-veterans-affairs/va.gov-team/31428).

Requirements for inbound and status as of 12NOV21:

|Status                    |ID        |Inbound Scenarios|CSP      |Login From|End   |Level of Authority|CSP Designation|Can they login to va.gov successfully?|Questions|Answers|Notes|Test account to be used for this validation|
|--------------------------|----------|-----------------|---------|----------|------|------------------|---------------|--------------------------------------|---------|-------|-----|-------------------------------------------|
|Done                      |inbound-3 |standard         |idme     |accessva  |va.gov|LOA3              |               |yes                                   |         |       |     |vets.gov.user+24@gmail.com                 |
|Done (confirmation needed)|inbound-10|standard         |mhv      |accessva  |va.gov|LOA2              |MHV Premium    |yes                                   |         |       |     |vets250                                    |
|Done                      |inbound-18|standard         |dslogon  |accessva  |va.gov|LOA2              |dslogon premium|yes                                   |         |       |     |ace.a.mcghee1                              |
|Done (confirmation needed)|inbound-21|standard         |dslogon  |ebenefits |va.gov|LOA2              |dslogon premium|yes                                   |         |       |     |ace.a.mcghee1                              |
|                          |inbound-73|lack of idme uuid|login.gov|accessva  |va.gov|LOA1              |               |yes                                   |         |       |     |joe.niquette+lgovial1@oddball.io           |
|                          |inbound-86|lack of idme uuid|login.gov|accessva  |va.gov|LOA3              |               |yes                                   |         |       |     |joe.niquette+lgovial2@oddball.io           |

**Comment by nsoutouras** [comment]
MSI - Joe will attempt to tackle.  Should finish this sprint.

**Comment by joeniquette** [comment]
https://docs.google.com/spreadsheets/d/1QhfzXDe6674oZ_D7EnM3uL3F3x580zF0c90d0qFdWdY/edit?pli=1#gid=120405345

**Comment by joeniquette** [comment]
must complete before 12/8

**Comment by joeniquette** [comment]
We are shifting from using codebuild as the infrastructure to run the tests over to using github actions with self hosted runners as per Ops recommendation here: https://dsva.slack.com/archives/CBU0KDSB1/p1638811784491900 and here: https://dsva.slack.com/archives/CBU0KDSB1/p1638989451106800.

**Comment by nsoutouras** [comment]
Nick: See #32381

**Comment by joeniquette** [comment]
this will be done before putting down login.gov blockers resolved

**Comment by nsoutouras** [comment]
Planning 1/31/22: We are stuck on this, move to backlog if no progress next monday.

**Comment by jscanish** [comment]
After talking with @joeniquette we've decided to close this ticket and rely instead on a manual Datadog process. Setting up this automated testing is proving far more difficult than we anticipated, and at this point the additional effort to get this working is just too great to be worth the time it will take.

And in the event that we do get it working properly, there is a real concern that keeping the tests up to date so they continue to pass will be a lot of continued effort.

The manual Datadog process is less ideal but it's still functional enough to render the extra work required here unnecessary.
